org.ghidra_sre.Ghidra.metainfo.xml

```diff
@@ -26,14 +28,13 @@
     <p>NOTE: This wrapper is not verified by, affiliated with, or supported by the National Security Agency Research Directorate.</p>
   </description>
   <url type="homepage">https://ghidra-sre.org/</url>
+  <url type="bugtracker">https://github.com/NationalSecurityAgency/ghidra/issues</url>
+  <url type="faq">https://github.com/NationalSecurityAgency/ghidra/wiki/Frequently-asked-questions</url>
+  <url type="help">https://htmlpreview.github.io/?https://github.com/NationalSecurityAgency/ghidra/blob/stable/GhidraDocs/CheatSheet.html</url>
+  <url type="vcs-browser">https://github.com/NationalSecurityAgency/ghidra</url>
+  <url type="contribute">https://github.com/NationalSecurityAgency/ghidra/blob/master/CONTRIBUTING.md</url>
   <screenshots>
     <screenshot type="default">
-      <caption>“Project” Window</caption>
-      <image width="805" height="605">
-        https://github.com/flathub/org.ghidra_sre.Ghidra/raw/master/screenshots/ghidra-project-view.png
```

Comment:
From https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-screenshots :
> Software centers displaying the component will usually prefer screenshots of the current environment and style, and display them first, even before the screenshot marked as default.

No need to reorder the tags here.